HopsanCLI/ModelUtilities.cpp

```diff
@@ -43,6 +40,19 @@
 using namespace std;
 using namespace hopsan;
 
+namespace {
+
+bool findAndRemove(std::vector<std::string>& rVector, const std::string& value) {
```

Comment:
As I understand it, this function removes the variable from the filter vector, and returns true if the variable was found. Why is it necessary to remove the variable? Is it not enough to check if the filter vector contains the variable? I guess there is a good reason, but at least adding some comments would be helpful.

Reply:
Yes, I will add some comments.
I remove the found name  so that the following searches will go faster. Probably no noticeable speed difference (especially for short lists), but just felt that it was unnecessary to keep it after it has been found since they will never be found again.

Reply:
Never mind, I changed it to a simple contains() function, that only looks for the data and doesn't remove it from the includeFilter, also takes includeFilter as const& as it wont need to change anymore. Should be more clear and robust.